Sources/PrimerSDK/Classes/Core/3DS/ThreeDSecureService.swift

```diff
@@ -0,0 +1,207 @@
+//
+//  ThreeDService.swift
+//  PrimerSDK
+//
+//  Created by Evangelos Pittas on 1/4/21.
+//
+
+#if canImport(UIKit)
+
+import Foundation
+import ThreeDS_SDK
+
+protocol ThreeDSecureServiceProtocol {
```

Comment:
[Q]
What's the point of this abstraction if only one function is defined here?
Couldn't you put more thing here?
`initializeSDK` or `performChallenge`?

Reply:
Agreed